client/src/config/ApolloClient.js

```diff
@@ -41,7 +41,7 @@ const link = from([
     uri:
       process.env.NODE_ENV === 'production'
         ? 'https://mm.dashnet.in/api/v1/graph'
-        : 'https://project-reclamation-staging.herokuapp.com/v1/graph',
+        : 'https://mm.dashnet.in/api/v1/graph',
```

Comment:
Do not use production systems in staging/development.

```suggestion
        : 'https://project-reclamation-staging.herokuapp.com/api/v1/graph',
```